must-gather/collection-scripts/gather

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# generate /must-gather/version file
+. version
+echo "node-maintenance-operator/must-gather" > /must-gather/version
+version >> /must-gather/version
+
+# get namespace of node-maintenance-operator
+NMO_NAMESPACE=$( oc get pods -A -l name='node-maintenance-operator' -o=jsonpath='{.items[0].metadata.namespace}{"\n"}' )
+# fallback to default
+[ -z "${NMO_NAMESPACE}" ] && ns="openshift-node-maintenance"
+
+# resource list
+resources=()
+
+# operator namespace
+resources+=(ns/${NMO_NAMESPACE})
+
+# node maintenance CRs
+resources+=(nodemaintenance)
```

Comment:
I think we should also collect Nodes and Machines
Or are they automatically collected by some base level functionality?